Traceback (most recent call last):
  File "train_celeba.py", line 163, in <module>
    main()
  File "train_celeba.py", line 78, in main
    dataset = dset.ImageFolder(root=args.celeba_path,
  File "/zhome/77/8/118225/.local/lib/python3.8/site-packages/torchvision/datasets/folder.py", line 226, in __init__
    super(ImageFolder, self).__init__(root, loader, IMG_EXTENSIONS if is_valid_file is None else None,
  File "/zhome/77/8/118225/.local/lib/python3.8/site-packages/torchvision/datasets/folder.py", line 114, in __init__
    raise RuntimeError(msg)
RuntimeError: Found 0 files in subfolders of: ../../Data/CelebA/celeba/Img
Supported extensions are: .jpg,.jpeg,.png,.ppm,.bmp,.pgm,.tif,.tiff,.webp
